src/plugins/teleport-plugin-vue-base-component/utils.ts

```diff
@@ -70,23 +71,40 @@ export const generateVueNodesTree = (
         return
       }
 
-      if (child.type === 'state') {
+      if (child.type === 'state' && child.states) {
```

Comment:
Added this check to save ``0.33`` sec when there are no states declared, not sure if it is actually needed 😅

Reply:
that's very specific :D